### PR TITLE
Addition of .gitignore

### PR DESCRIPTION
**Pull request by Nelson1211**

### CODE REVIEW

File: home/migrations/0001_initial.py [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+                ('from_account', models.IntegerField()),
+                ('to_account', models.IntegerField()),
+                ('transaction_value', models.BigIntegerField()),
+                ('transaction_date', models.DateTimeField()),
+                ('transaction_type', models.CharField(max_length=20)),
+                ('user_id', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to='home.User')),
+            ],
+        ),
+        migrations.CreateModel(
+            name='Requests',
```
Comment (vijaiharihar) [inline home/migrations/0001_initial.py]: Don't we need from account, to account and amount fields here? How are we handling transaction requests like critical transactions?

File: home/models.py [hunk 2]
```diff
@@ -1,3 +1,84 @@
...
+    password = models.CharField(max_length=200)
+    privilege_id = models.ForeignKey(Privilege, on_delete=models.CASCADE)
+
+    def __str__(self):
+        if self.first_name:
+            return self.first_name
+        else:
+            return self.business_name
+
+class Account(models.Model):
```
Comment (vijaiharihar) [inline home/models.py]: How are credit cards handled? Don't we need stuff like credit card number and so on? Probably we should check with TA and commit this for now?

Comment (majluni) [reply home/models.py]: I do not believe we need to have actual cards. What I see in the document is talking about debiting or crediting an account. Meaning, just moving money in or out.

File: home/migrations/0001_initial.py (hunk 1 above)
Comment (Nelson1211) [reply home/migrations/0001_initial.py]: Check the models.py file

**Pull request merged**